syntax = "proto3";

package dbaccessor;

option go_package = "grpc_chat/dbaccessor";

message Message {
  string user_id = 1;
  string text = 2;
}

message SimpleResponse {
  bool responseFlag = 1;
}

service dbaccessor {
  rpc insertMessage(Message) returns (SimpleResponse) {};
}
